qa/workunits/ceph-disk/ceph-disk-test.py

```diff
@@ -193,7 +177,39 @@ def setup(self):
                 del c.conf['global'][key]
         c.save_conf()
 
-    def test_destroy_osd(self):
+    def test_deactivate_reactivate_osd(self):
+        c = CephDisk()
+        have_journal=True
+        disk = c.unused_disks()[0]
+        osd_uuid = str(uuid.uuid1())
+        c.sh("ceph-disk zap " + disk)
+        c.sh("ceph-disk prepare --osd-uuid " + osd_uuid +
+             " " + disk)
+        c.wait_for_osd_up(osd_uuid)
+        device = json.loads(c.sh("ceph-disk list --format json " + disk))[0]
+        assert len(device['partitions']) == 2
+        c.check_osd_status(osd_uuid, have_journal)
+        data_partition = c.get_osd_partition(osd_uuid)
+        c.sh("ceph-disk deactivate " + data_partition['path'])
```

Comment:
<pre>
c.wait_for_osd_down(osd_uuid)
</pre>

The idea is to verify at least one side effect of deactivate and this seem the simplest way. At least for me, feel free to verify something else if you prefer.